src/Deserializers/LegacyPropertyDeserializer.php

```diff
@@ -95,10 +72,23 @@ private function getPropertyId() {
 	}
 
 	/**
-	 * @return Fingerprint
+	 * @param array $serialization
+	 * @return string
```

Comment:
Is there a specification for PHPDoc to tell which documentation is "valid"? Imo, it doesn't make sense to add `@throws` on private methods because the class itself knows which exceptions the code throws.